stories/Documentation/ComponentStructure.mdx

```diff
@@ -0,0 +1,296 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta
+  title="Getting started/Component Structure"
+  parameters={{
+    viewMode: "docs",
+    previewTabs: {
+      canvas: { hidden: true }
+    }
+  }}
+/>
+
+# Component Structure and Standards
+
+This guide explains the structure and standards used across the Mangrove component library to help you understand and work with existing components.
+
+## Component Architecture
+
+The Mangrove library is organized using a loosely atomic design approach, with components categorized by their function and complexity.
+
+### Directory Structure
+
+```
+stories/
+├── Components/           # All reusable components
+│   ├── Accordion/        # Component category
+│   ├── Buttons/          # Component category
+│   │   ├── CtaButton/    # Individual component
+│   │   │   ├── CtaButton.jsx
+│   │   │   ├── CtaButton.mdx
+│   │   │   ├── CtaButton.stories.jsx
+│   │   │   └── buttons.scss
+│   │   └── ...
+│   └── ...
+├── Documentation/        # Documentation files
+├── assets/               # Shared assets
+│   ├── scss/             # Global SCSS files
+│   │   ├── variables.scss
+│   │   ├── mixins.scss
+│   │   └── ...
+│   ├── fonts/
+│   └── images/
+└── __tests__/            # Test files
+```
+
+The jest test files can be created at different levels in the project structure depending on the scope of the tests.
+
+## Component File Structure
+
+Each component in the library consists of 3-4 files:
+
+1. **ComponentName.jsx** - The React component implementation
+2. **ComponentName.mdx** - Documentation for the component
+3. **ComponentName.stories.jsx** - Storybook stories showcasing the component
+4. **component-name.scss** - Component-specific styles (when needed)
+
+### Component Implementation (.jsx)
+
+The React component file (.jsx) follows these conventions:
+
+- Functional components with React hooks
+- Props destructuring with appropriate defaults
+- Accessibility attributes where needed
+- Semantic HTML elements
+- CSS class prefixing with `mg-`
+
+Example structure:
+
+```jsx
+import React from 'react';
+// Import component-specific styles if needed
+// import './component-name.scss';
```

Comment:
Instead let's use this to say we shouldn't import Sass directly to the component but to add the sass import at https://github.com/unisdr/undrr-mangrove/blob/main/stories/assets/scss/_components.scss